lisa/testsuite.py

```diff
@@ -40,6 +41,16 @@
 )
 from lisa.util.perf_timer import Timer, create_timer
 
+TestSuiteStatus = Enum(
```

Comment:
It doesn't worth to create suite message in LISA. In the default lisa runner, each test case is scheduled individually. So you can consider the test suite will be start as many as the test case method. So it's status is same as test result status.
If Junit needs this, we can use test result message to simulate it.